Add more sensors to System monitor (#40578)
home-assistant/home-assistant.github.io: source/_integrations/systemmonitor.markdown

### Disks

One sensor per discovered disk/mount point will be created

- **Disk free**: Amount of free space on the disk
- **Disk use**: Amount of used space on the disk
- **Disk usage (%)**: Percentage of disk space used

### Network

One sensor per discovered network interface will be created

- **IPv4 address**: The IPv4 address assigned to the network interface
- **IPv6 address**: The IPv6 address assigned to the network interface
- **Network in**: Total bytes received on the network interface
- **Network out**: Total bytes sent from the network interface
- **Packets in**: Number of packets received on the network interface
- **Packets out**: Number of packets sent from the network interface
- **Network throughput in**: Current inbound network speed (bytes per second)
- **Network throughput out**: Current outbound network speed (bytes per second)

### Other

- **Battery**: Percentage of battery remaining

> - **Battery empty**: Expected time until the battery is empty if not plugged in

Reviewer: ```suggestion
- **Battery time remaining**: Expected time until the battery is empty if not plugged in
```

Author: This is my mistake. It's a timestamp when the battery is empty so the entity name is still `Battery empty` but the description needs to change reflecting it's the end time of the battery.

> - **Battery empty**: Expected time until the battery is empty if not plugged in

Author: ```suggestion
- **Battery empty**: Expected time when the battery is empty if not plugged in
```

- **Charging**: Battery is charging (binary sensor)
- **Fan speed**: Built-in fan speeds
- **Last boot**: The date and time when the system was last started
- **Load (1m)**: System load average over the last 1 minute
- **Load (5m)**: System load average over the last 5 minutes

> - **Load (15m)**: System load average over the last 15 minutes

Reviewer: ```suggestion
- **Load (1 min)**: System load average over the last 1 minute
- **Load (5 min)**: System load average over the last 5 minutes
- **Load (15 min)**: System load average over the last 15 minutes
```

- **Memory free**: Amount of available system memory
- **Memory use**: Amount of used system memory

> - **Memory usage (percent)**: Percentage of system memory used

Reviewer: ```suggestion
- **Memory in use (GB)**: Amount of system memory used
- **Memory in use (%)**: Percentage of system memory used
```
Can the terms "use" and "usage" be streamlined?

Author: No, because they refer to how the entities are named in core so this needs to remain as this. Description could be adjusted.

> - **Memory usage (percent)**: Percentage of system memory used

Author: ```suggestion
- **Memory use**: Amount of system memory used
- **Memory usage (%)**: Percentage of system memory used
```

- **Processor use**: Percentage of CPU usage
- **Processor temperature**: Current temperature of the processor
- **Swap free**: Amount of available swap memory
- **Swap use**: Amount of used swap memory

> - **Swap usage (percent)**: Percentage of swap memory used

Reviewer: ```suggestion
- **Swap usage (%)**: Percentage of swap memory used
```
this also?

### Disks

One sensor per discovered disk/mount point will be created

| Sensor type      | Type   | Description                                  |
|------------------|--------|----------------------------------------------|
| Disk free        | sensor | Amount of free space on the disk             |
| Disk use         | sensor | Amount of used space on the disk             |
| Disk usage (%)   | sensor | Percentage of disk space used                |

### Network

One sensor per discovered network interface will be created

> | Sensor                | Type   | Description                                         |

Reviewer: It already says Sensor in the title. Could the type be skipped here? They are all sensors. Then this could just be a list. 
**IPv4 address**: The IPv4 address assigned ...

|-----------------------|--------|-----------------------------------------------------|
| IPv4 address          | sensor | The IPv4 address assigned to the network interface  |
| IPv6 address          | sensor | The IPv6 address assigned to the network interface  |
| Network in            | sensor | Total bytes received on the network interface       |
| Network out           | sensor | Total bytes sent from the network interface         |
| Packets in            | sensor | Number of packets received on the network interface |
| Packets out           | sensor | Number of packets sent from the network interface   |
| Network throughput in | sensor | Current inbound network speed (bytes per second)    |
| Network throughput out| sensor | Current outbound network speed (bytes per second)   |

### Other

> | Sensor                 |  Type         | Description                                               |

Reviewer: same here, it says Sensor in the title. They are all sensors. Maybe just add binary sensor in brackets, and make it a list instead of a table? And mark up the name bold, to make it clear it is the UI string that is shown.
**Battery**: Percentage of battery remaining
